src/main/resources/com/parasoft/findings/jenkins/coverage/api/metrics/model/Messages.properties

```diff
@@ -49,5 +49,5 @@ Metric.Coverage.Missed=Missed
 Baseline.PROJECT=Overall project
 Baseline.MODIFIED_LINES=Modified code lines
 
-Criticality.UNSTABLE=Set the build status to unstable if the quality gate has been missed
-Criticality.FAILURE=Fail the step if the quality gate has been missed
\ No newline at end of file
+Criticality.UNSTABLE=Set the build status to unstable if the quality gate fails
+Criticality.FAILURE=Fail the step if the quality gate fails
```

Comment:
I don't see the value to keep the original phrases as "Fail the step ..." here. 
Consistency might be better (Set the build status to Failure if the the quality gate fails). 
What do you think @rwang-parasoft ?

Reply:
Remove the options help description.